src/api/tiles.ts

```diff
@@ -170,18 +176,6 @@ function createTilesApi({
           'INSERT INTO TileData (tileHash, tilesetId, data) VALUES (:tileHash, :tilesetId, :data)'
         ).run({ tileHash, tilesetId, data })
 
-        // TODO: Is this still necessary?
-        db.prepare<{
-          etag?: string
-          tilesetId: string
-          upstreamUrl?: string
-        }>(
-          'UPDATE Tileset SET (upstreamUrl) = (:upstreamUrl) WHERE id = :tilesetId'
-        ).run({
-          tilesetId,
-          upstreamUrl: upstreamTileUrl,
-        })
```

Comment:
pretty sure this would've caused some wild bugs - think at the time, i was confused about how upstream urls worked in the context of the tilejson spec 😅